packages/rpc/src/common.ts

```diff
@@ -104,10 +104,10 @@ export type RpcRequest = WithHash<{
     destinationAsset: UncheckedAssetAndChain,
     destinationAddress: string,
     brokerCommission: number,
-    ccmMetadata?: Nullish<CcmParams>,
-    boostFee?: Nullish<number>,
-    affiliateFees?: Nullish<{ account: string; bps: number }[]>,
-    fillOrKillParams?: FillOrKillParams,
+    ccmMetadata: Nullish<CcmParams>,
+    boostFee: Nullish<number>,
+    affiliateFees: Nullish<{ account: string; bps: number }[]>,
+    fillOrKillParams: FillOrKillParams,
```

Comment:
forgot to remove the `?` in https://github.com/chainflip-io/chainflip-product-toolkit/pull/319 (thanks @GabrielBuragev)